README.md

```diff
@@ -1 +1,93 @@
-# Stable-diffusion-3
\ No newline at end of file
+# Stable-Diffusion-3
+This is a stable-diffusion-3 template you can use to import your model on Inferless Platform.
+
+---
+## Prerequisites
+- **Git**. You would need git installed on your system if you wish to customize the repo after forking.
+- **Python>=3.8**. You would need Python to customize the code in the app.py according to your needs.
+- **Curl**. You would need Curl if you want to make API calls from the terminal itself.
+
+---
+## Quick Start
+Here is a quick start to help you get up and running with this template on Inferless.
+
+### Fork the Repository
+Get started by forking the repository. You can do this by clicking on the fork button in the top right corner of the repository page.
+
+This will create a copy of the repository in your own GitHub account, allowing you to make changes and customize it according to your needs.
+
+## Create a Custom Runtime in Inferless
+To access the custom runtime window in Inferless, simply navigate to the sidebar and click on the **Create new Runtime** button. A pop-up will appear.
+
+Next, provide a suitable name for your custom runtime and proceed by uploading the **config.yaml** file given above. Finally, ensure you save your changes by clicking on the save button.
+
+
+### Import the Model in Inferless
+Log in to your inferless account, select the workspace you want the model to be imported into and click the Add Model button.
+
+Select the PyTorch as framework and choose **Repo(custom code)** as your model source and use the forked repo URL as the **Model URL**.
+
+After the create model step, while setting the configuration for the model make sure to select the appropriate runtime.
+
+Enter all the required details to Import your model. Refer [this link](https://docs.inferless.com/integrations/github-custom-code) for more information on model import.
+
+The following is a sample Input and Output JSON for this model which you can use while importing this model on Inferless.
```

Comment:
You have written input and output json , but given only Input json

Reply:
Hey @nickaggarwal,
Please check I have updated the page.

Reply:
Hey @nickaggarwal,
Please check I have updated the page.